kytos/core/switch.py

```diff
@@ -191,6 +194,25 @@ def as_json(self):
         return json.dumps(self.as_dict())
 
 
+class CONNECTION_STATE(Enum):
```

Comment:
move `CONNECTION_STATE`, `ConnectionProtocol` and `Connection` to a new module , named `connection`